src/Microsoft.Win32.Registry/src/Microsoft.Win32.Registry.csproj

```diff
@@ -31,7 +31,7 @@
     <Compile Include="Microsoft\Win32\SafeHandles\SafeRegistryHandle.cs" />
     <Compile Include="System\Security\AccessControl\RegistryRights.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp' AND '$(TargetsWindows)' == 'true'">
+  <ItemGroup Condition="('$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'uap') AND '$(TargetsWindows)' == 'true'">
```

Comment:
I agree it would be cleaner, but @weshaggard suggested that we should always do inclusive conditions as opposed to exclusive, I'm fine with either.